### PR TITLE
[Admin Log] fix log (resend) + search onChange 

### PR DESCRIPTION
**Pull request by celynolivia**
Fix Admin Log Resend + Add Search onChange

### CODE REVIEW

**Review by ayusrina — APPROVED** [review]

File: src/pages/Customers.js [hunk 1]
```diff
@@ -21,74 +21,75 @@ import {
...
     setLoading(true);
     let customerToken = await getTokenCustomer(rowData.cust_email);
 
-    // var hasil = await insertLog(rowData.account_num,"RESEND");
-    // console.log(hasil)
+    let hasil = await insertLog(rowData.account_num,"RESEND");
 
-    if (true) {
+    console.log("ini hasil");
+    console.log(hasil)
```
Comment (sayaandreas) [inline src/pages/Customers.js]: nitip celyn

File: src/pages/Customers.js [hunk 2]
```diff
@@ -21,74 +21,75 @@ import {
...
+    if(hasil){
+      console.log("masuk sini");
```
Comment (sayaandreas) [inline src/pages/Customers.js]: sekalian

File: src/pages/Customers.js (hunk 1 above)
Comment (celynolivia) [reply src/pages/Customers.js]: thank youu andreass :D

**Review by sayaandreas — APPROVED** [review]
LGTM

**Pull request merged**